.github/workflows/build-lint-test.yml

```diff
@@ -30,13 +44,18 @@ jobs:
             ~/.cargo/registry/index/
             ~/.cargo/registry/cache/
             ~/.cargo/git/db/
-            target/
+            ./backend/target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: ${{ runner.os }}-cargo-
       - name: Install sqlx-cli
-        run: cargo install sqlx-cli --no-default-features --features sqlite
+        run: cargo --verbose --locked install sqlx-cli --no-default-features --features sqlite
       - name: Create database and run migrations
         run: sqlx database setup
+      - name: Download frontend-build
+        uses: actions/download-artifact@v4
+        with:
+          name: frontend-build
+          path: ../frontend/dist
```

Comment:
The frontend is already built in this job, so no need to download it from the other job

Reply:
Yes, spotted it as wel, fixed in [1c08cc2](https://github.com/kiesraad/abacus/pull/429/commits/1c08cc27244039b43e4c870bd9c4da4b50b7cef0)